workspaces/tech-radar/plugins/tech-radar/README.md

```diff
@@ -283,6 +283,19 @@ export const app = createApp({
 });
 ```
 
+#### Option C: Opt-out of statefullness and manage state independently
+
+In situations where you need to integrate a radar and manage state independently of Backstage, use the stateless UI component:
```

Comment:
it's already possible to do that. Have you tried providing your own implementation of `techRadarApiRef` by adding the following to `apis.ts`?

```ts
   createApiFactory({
      api: techRadarApiRef,
      deps: {},
      factory: () =>
        new (class MyApi implements TechRadarApi {
          async load(): Promise<TechRadarLoaderResponse> {
            return { entries: [], quadrants: [], rings: [] };
          }
        })(),
    }),
```

Reply:
My use case does not use Backstage though I would like to take advantage of the amazing component that is written in this plugin. In such cases unless I am misunderstanding I don't believe `createApiFactory` will be available, and it will be up to the upstream consumer to manage state for the radar chart.

Reply:
in your Backstage app you should have a `packages/app/src/apis.ts` file where you can add the code above that loads the data from whatever data source you want. The data will be injected into tech-radar plugin once loaded.

Reply:
I see! Thx for clarifying @harley-davidson, it makes sense 👍

Reply:
in that case I'm not sure we need to explain this in the README, but calling @awanlin as he can have a better opinion on this

Reply:
Thanks @vinzscam ! @awanlin , thoughts on how to best get this one merged?

Reply:
Appreciate that @awanlin - I imagine the integrity of the plugin is more important than the underlying UI. On our end we could look to develop this type of component independently, but to offer a counterpoint, I don't think that the impact of exporting a stateless component would be a huge risk. Even if the API changed, upstream consumers could just pin to a version of the plugin until they have capacity to upgrade. However, I do understand that you want the focus to be on the plugin and the UI's usage within Backstage itself.

Reply:
What are your thoughts @vinzscam ?